egs/heroico/s5/README.md

```diff
@@ -0,0 +1,28 @@
+# heroico
+Scripts  for Kaldi recipe for Spanish using the Heroico corpus.
+
+The Heroico corpus includes a set of recordings from nonnative speakers.
+
+WER Results:
```

Comment:
Hi John, could you please split this file into two -- README and RESULTS, please?